orangecontrib/text/widgets/owpreprocess.py

```diff
@@ -401,6 +401,8 @@ def get_value(self):
             checked.pop(checked.index(self.KEEP_N))
             if self.FREQUENCY not in self.checked:
                 checked.append(self.FREQUENCY)
+                self.frequency_filter.min_df = 0
+                self.frequency_filter.max_df = 1.0
```

Comment:
Could we move these two lines up? So we have code for frequency first and then code for keep_n.
IMO something like this is easier to understand:
```py
if self.FREQUENCY in self.checked:
    ...
else:
    ...

if self.KEEP_N in self.checked:
    ...
else:
    ...
```